pkg/features/kube_features.go

```diff
@@ -1009,7 +1009,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	genericfeatures.APIPriorityAndFairness:              {Default: true, PreRelease: featuregate.Beta},
 	genericfeatures.OpenAPIEnums:                        {Default: true, PreRelease: featuregate.Beta},
 	genericfeatures.CustomResourceValidationExpressions: {Default: false, PreRelease: featuregate.Alpha},
-	genericfeatures.OpenAPIV3:                           {Default: false, PreRelease: featuregate.Alpha},
+	genericfeatures.OpenAPIV3:                           {Default: true, PreRelease: featuregate.Beta},
```

Comment:
I don't think we are defaulting beta features on anymore?

Reply:
Although [Jordan seems to think the rule doesn't apply to this](https://kubernetes.slack.com/archives/C0EG7JC6T/p1647964781225229?thread_ts=1647954978.154789&cid=C0EG7JC6T), I'm not so sure -- e.g. are you 100% sure you're not going to change the url structure for GA?o

Reply:
/cc @apelisse 

We're planning on keeping the same interface for GA

Reply:
It's gotta be more than just a plan! Imagine a v1.25 client using openapi v3 by default which happens to talk to a 1.24 cluster--

Reply:
OK, that's good enough for me!